config/initializers/airbrake.rb

```diff
@@ -1,7 +1,17 @@
-if !defined?(Rails) || Rails.env.staging? || Rails.env.production?
-  require 'airbrake'
+require 'airbrake'
 
+if !defined?(Rails) || Rails.env.staging? || Rails.env.production?
   Airbrake.configure do |config|
     config.api_key = ENV['AIRBRAKE_API_KEY']
   end
+else
+  module Airbrake
```

Comment:
Hmm... you're right, I could have sworn the test helpers were using it. But nope, my mistake!

Made it a standalone dummy module, rather than monkey-patching.